includes/wikia/parser/templatetypes/tests/NavigationTemplateTest.php

```diff
@@ -35,7 +36,7 @@ public function getNavigationTemplates() {
 		return [
 			[
 				'<a>This is a <strong>template</strong> <b>without</b> a <span>block</span> element</a>.',
-				'<a>This is a <strong>template</strong> <b>without</b> a <span>block</span> element</a>.',
+				"\n<a>This is a <strong>template</strong> <b>without</b> a <span>block</span> element</a>.\n",
```

Comment:
Yes, we need to keep the output clean - if there's no removal, the markup has to stay intact